de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/ddl/functions/customerItemStatistics.sql

```diff
@@ -0,0 +1,74 @@
+DROP FUNCTION IF EXISTS customerItemStatistics(numeric, numeric, date, date, numeric, numeric);
+
+CREATE OR REPLACE FUNCTION customerItemStatistics(
+     p_AD_Client_ID numeric,
+     p_AD_Org_ID numeric,
+     p_dateFrom date, 
+	 p_dateTo date,
+	 p_C_BPartner_ID numeric, 
+	 p_M_Product_ID numeric)
+
+  RETURNS TABLE
+(
+
+	 BPValue character varying,
+	 productValue character varying,
+	 description character varying,
+	 qtyInvoiced numeric,
+	 uomSymbol  character varying,
+	 Amount numeric,
+	 ProductCosts numeric,
+	 ProductCostsPercent numeric
+ 
+) AS
+
+
+$$
+
+
+SELECT 
+	t.BPValue,
+	t.productValue,
+	t.productName,
+	t.qtyInvoiced,
+	t.uomSymbol,
+	t.Betrag as Amount,
+	t.costPrice * t.qtyInvoiced as ProductCosts,
+	CASE WHEN t.betrag > 0 THEN (t.costPrice * t.qtyInvoiced * 100 )/ t.betrag else 0 end as  ProductCostsPercent
```

Comment:
suggestion: round the procent to 2 decimals

Reply:
Done